### PR TITLE
Add initial Java documentation

### PR DESCRIPTION
**Pull request by nomisRev**
This PR adds an initial section of the Java docs. In subsequent PRs we should add documentation for pdf, and web searching, context, and a small section on how to work with Loom.

### CODE REVIEW

File: content/docs/learn/quickstart/java.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  </TabItem>
+  <TabItem value="gradle" label="Gradle">
+
+```kotlin
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  implementation("com.xebia:xef-core:<version>")
```
Comment (juanpedromoreno) [inline content/docs/learn/quickstart/java.md:41]:
```suggestion
  implementation("com.xebia:xef-java:<version>")
```

**Review by juanpedromoreno — DISMISSED** [review]
Thanks @nomisRev !

File: content/docs/learn/quickstart/java.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+in your build, if you haven't done it before. Then add the library in the usual way.
+
+<Tabs>
+  <TabItem value="maven" label="Maven" default>
+
+```xml
+<dependency>
+    <groupId>com.xebia</groupId>
+    <artifactId>xef-java</artifactId>
+    <version>0.2.0</version>
```
Comment (juanpedromoreno) [inline content/docs/learn/quickstart/java.md:27]:
```suggestion
    <version>0.0.2</version>
```

**Review by juanpedromoreno — APPROVED** [review]
Sorry, didn't see this earlier.

**Review by Yawolf — APPROVED** [review]

File: content/docs/learn/quickstart/java.md (hunk 2 above)
Comment (nomisRev) [reply content/docs/learn/quickstart/java.md]: Oh, we're still on patches. I thought we were bumping minors giving the new features.

**Review by juanpedromoreno — APPROVED** [review]

**Pull request merged**